lib/alexa.js

```diff
@@ -367,6 +367,9 @@ function ResponseBuilder(self) {
 
                 responseObject.response.directives = [audioPlayerDirective];
                 return this;
+            },
+            'setResponse': function () {
```

Comment:
Yeah, I think we are in favor of using `:ask`, `:tell` or `:delegate`. But don't know how many devs would want to use responseBuilder over these! If devs prefer ResponseBuilder then yes we can consider adding dialog directives!